starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -41,4 +42,12 @@ public List<Inventory> findInventories() {
   public Inventory saveInventory(@Valid @RequestBody Inventory inventory) {
     return this.inventoryDAO.create(inventory);
   }
+
+  @DeleteMapping
+  public void deleteInventoryById(@RequestBody List<String> ids) {
```

Comment:
You need to return something from this endpoint so the frontend knows if the operation was successful and which inventories were deleted. You can either return which ids were successfully deleted or a list of the inventories left.